tests/terraform/createcluster.go

```
@@ -0,0 +1,61 @@
+package e2e
+
+import (
+	"flag"
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+var destroy = flag.Bool("destroy", false, "a bool")
+var nodeOs = flag.String("node_os", "centos8", "a string")
+var externalDb = flag.String("external_db", "mysql", "a string")
+var arch = flag.String("arch", "amd64", "a string")
+var clusterType = flag.String("cluster_type", "etcd", "a string")
+var resourceName = flag.String("resource_name", "etcd", "a string")
+var sshuser = flag.String("sshuser", "ubuntu", "a string")
+var sshkey = flag.String("sshkey", "", "a string")
+
+var (
+	kubeConfigFile string
+	masterIPs      string
+	workerIPs      string
+)
+
+func BuildCluster(nodeOs, clusterType, externalDb, resourceName string, t *testing.T, destroy bool) (string, string, string) {
+
+	tDir := "./modules/k3scluster"
+	vDir := "/config/" + nodeOs + clusterType + ".tfvars"
+
+	if externalDb != "" {
+		vDir = "/config/" + nodeOs + externalDb + ".tfvars"
+	}
+
+	tfDir, _ := filepath.Abs(tDir)
```

Is it safe to be ignoring the errors here?